packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -16,9 +16,10 @@ const convert = require("convert-units");
 
 
 const PERSONAL_RELATIONSHIP_TYPE_CODE = "2.16.840.1.113883.1.11.19563";
-const decimal_regex = /-?(?:0|[1-9][0-9]*)(?:\.[0-9]+)?(?:[eE][+-]?[0-9]+)?/;
+const decimal_regex = /-?(?:(?:0|[1-9][0-9]*)\.?[0-9]*|\.[0-9]+)(?:[eE][+-]?[0-9]+)?/;
```

Comment:
we were missing decimals without a leading interger, so 
.5 was no good but 0.5 was. Patch for that